packages/external-db-authorization/lib/factory.js

```diff
@@ -1,37 +1,33 @@
-const strategyFor = (vendor = '', cfg) => {
+const authProviderFor = (vendor = '', cfg) => {
   switch (vendor.toLowerCase()) {
     case 'aws': {
       const { AwsStrategy } = require('./strategies/aws_strategy')
-      
-      return { authStrategy: new AwsStrategy(cfg), isValidAuthService: true }
-    
+      return new AwsStrategy(cfg)
     }
     case 'gcp': {
       const { GcpStrategy } = require('./strategies/gcp_strategy')
-
-      return { authStrategy: new GcpStrategy(cfg), isValidAuthService: true } 
+      return new GcpStrategy(cfg)
     }
 
     case 'azure':
     case 'local':
     default: {
       const { LocalStrategy } = require('./strategies/local_strategy')
       
-      return { authStrategy: new LocalStrategy(cfg), isValidAuthService: false }  
+      return { authProvider: new LocalStrategy(cfg), isValidAuthService: false }  
     }      
   }
 }
 
-const createAuthService = async(vendor, config) => {
+const initAuthProvider = async(vendor, config) => {
 
   const { auth: authConfig } = await config.readConfig()
-  const { validAuthConfig } = await config.configStatus()
+  const authInformation = await config.configStatus()
+
+  const authProvider = authProviderFor(vendor, authConfig)
```

Comment:
If the case was default (local/azure) , so you receive an object back from that function, how do you handle this response? I don't find it in the code